app/styles/_conversation.sass

```diff
@@ -158,13 +160,16 @@ $conversation-typing-height: 0.6rem
       text-align: left
       float: left
     span.content
-      border-radius: 0.5rem 0 0 0.5rem
+      border-radius: 1.3rem 0 1.3rem 1.3rem
       background: #fff
+      margin-right: 0.7rem
+      color: rgb(53, 53, 53)
+      margin-left: 0rem
     time
       float: right
       text-align: right
       position: relative
-      right: 0.2rem
+      right: 0.9rem
```

Comment:
in https://github.com/mozillahispano/openwapp/commit/03584f00f46b05e2a7aa6f71c3679a94ea4647a5#diff-d4fd7d2b2ae3d3b687a2c646a3edcea0 i removed these lines and moved the time in the bubble. We can choose to revert that....